applications/ecallmgr/src/ecallmgr_call_events.erl

```diff
@@ -1086,6 +1086,12 @@ get_billing_seconds(Props) ->
         Billmsec -> kz_term:to_binary(kz_term:ceiling(Billmsec / 1000))
     end.
 
+-spec get_ringing_seconds(kz_term:proplist()) -> kz_term:api_binary().
```

Comment:
Return should be `kz_term:ne_binary()` since `'undefined'` can't be returned.

Reply:
Ah ok. Done.